src/beyondllm/llms/cohere.py

```diff
@@ -0,0 +1,60 @@
+from beyondllm.llms.base import BaseLLMModel, ModelConfig
+from typing import Any, Dict
+from dataclasses import dataclass, field
+import os
+import cohere
```

Comment:
import the cohere module inside ``load_llm`` refer to other LLMs. This needs to be under exception block